osquery/sql/sqlite_util.cpp

```diff
@@ -30,6 +30,11 @@ FLAG(string,
      "Not Specified",
      "Comma-delimited list of table names to be disabled");
 
+FLAG(string,
+     enable_tables,
+     "Not Specified",
```

Comment:
I think there is a way to check if this value is supplied or not. This means we can keep the default to `""` instead of a well-known canary value.

Reply:
Done